api.js

```diff
@@ -209,4 +209,53 @@ exports.setApp = function (app, db) {
       res.status(200).json({ error: err.message });
     }
   });
+
+  //--------------------Create Survey API--------------------//
+  app.post("/api/createSurvey", async (req, res, next) => {
+    // incoming: 
+    // outgoing: 
+
+    const { 
+      title_survey,
+      description_survey,
+      participants_emails,
+      start_survey,
+      end_survey,
+      questions_survey,
+      type1_questions,
+      type2_questions
+    } = req.body;
+    let valid = true;
+
+
+    if (valid) {
+      const result = db.collection("Survey").insertOne(
+        {
+            title_survey: title_survey,
+            description_survey: description_survey,
+            participants_emails: participants_emails,
+            start_survey: start_survey,
+            end_survey: end_survey,
+            questions_survey: questions_survey,
+            type1_questions: type1_questions,
```

Comment:
The questions should be inserted into a different collection. It's called "Questions". The format for the questions is as follows:
```
{
  _id: ObjectId
  type: "rating" or "text"
  title: string (title of the question)
  question_num: int (tells the program the order that the question was inserted in)
}
```